src/Configuration/Definition/ConfigRowDefinition.php

```diff
@@ -46,6 +43,7 @@ public function getParametersDefinition(): ArrayNodeDefinition
                 ->integerNode('retries')
                     ->min(0)
                 ->end()
+                ->booleanNode('advancedMode')->end()
```

Comment:
to je na co? to se objevuje nekde v configu?

Reply:
Souvisi to s odpovedi 
https://github.com/keboola/db-extractor-common/pull/94/files#r226447144

Resil bych to v separatnim issue.

Reply:
advancedMode jsem v masteru prave nenasel, proto se tak blbe ptam

Reply:
To je specialita minimalne tady https://github.com/keboola/db-extractor-mysql/blob/master/src/Keboola/DbExtractor/Configuration/MySQLConfigDefinition.php#L62

Prislo mi jednodussi pridat to i sem, ale asi to nema moc velky vyznam. Jednak se to da imho vyresit i jinak a za druhe kdyz tam budeme mit `ignoreExtraKeys()`, tak to tam ani potreba nebude kvuli kompatibilite.

Reply:
jo, je to hack, protoze byl chyba na front-end a poslil tu.  byl to nejednodusi ho dat tady a ignorovat.